webapp/app/css/treeherder.css

```diff
@@ -1041,10 +1037,6 @@ ul.failure-summary-list li .btn-xs {
     padding: 1px 2px 1px 2px;
 }
 
-.btn-icon-top-margin {
-    margin-top: 0.3em;
-}
-
```

Comment:
These two css entries are only used by the save cluster, so are safe for removal as far as I can tell.